frontend/src/app/modules/imports/services/data.service.ts

```diff
@@ -17,21 +18,30 @@ import {
   ContentMappingValues,
 } from '../models/mapping.model';
 import { ConfigService } from '@geonature/services/config.service';
+import { ActivatedRoute } from '@angular/router';
 
 @Injectable()
 export class DataService {
   private urlApi = null;
+  private destination: string = null;
 
   constructor(
     private _http: HttpClient,
     public config: ConfigService
   ) {
-    this.urlApi = `${this.config.API_ENDPOINT}/import/synthese`;
-    // this.urlApi = `${this.config.API_ENDPOINT}/import/occhab`;
+    this.urlApi = `${this.config.API_ENDPOINT}/import`;
+  }
+
+  setDestination(destination: string) {
+    this.destination = destination;
+  }
+
+  getUrlApi() {
+    return `${this.urlApi}/${this.destination}`;
   }
```

Comment:
**Suggestion:** It's a good practice to encapsulate the logic of creating URLs in a separate method. This will make the code more maintainable and easier to read. [best practice]
```suggestion
  getUrlApi(endpoint: string) {
      return `${this.urlApi}/${this.destination}/${endpoint}`;
  }
```